src/AbstractTrackingListener.php

```diff
@@ -48,6 +48,7 @@ public function getSubscribedEvents()
             'prePersist',
             'onFlush',
             'loadClassMetadata',
+            'preRemove',
```

Comment:
Adding the subscribed event here means that the IP traceable and timestampable listeners are also getting "on remove" support.  Is this intended?

Reply:
Makes sense, and no that was necessarily intentional.  Is that an issue though?

Reply:
It could be, as this change implies all extensions (the ones in this library and in the off chance someone wrote their own) using this listener need to support the Doctrine `preRemove` event.  But as is, there isn't a way to configure that for the other extensions.  So if the intent is to only add it to blameable, I'd just do a `public function getSubscribedEvents() { return array_merge(parent::getSubscribedEvents(), ['preRemove']; }` override to not have to worry about it.